src/subs/FitSubroutines.cpp

```diff
@@ -2234,17 +2235,17 @@ void Astro::saveResults(const astrometry::MCat &matches, string outCatalog, stri
             pmClip[i] = pmDetPtr->isClipped;
             pmReserve[i] = pmDetPtr->itsMatch->getReserved();
             vector<float> vv(5);
-            for (int i = 0; i < 5; i++) vv[i] = pmDetPtr->pmMean[i];
+            for (int ii = 0; ii < 5; ii++) vv[ii] = pmDetPtr->pmMean[ii];
             // Keep position in WCS_UNITS (degrees), put others into RESIDUAL_UNITS
-            vv[astrometry::VX] *= units[i];
-            vv[astrometry::VY] *= units[i];
-            vv[astrometry::PAR] *= units[i];
+            vv[astrometry::VX] *= units[astrometry::VX];
```

Comment:
Yes. I changed this because I saw that `i` was being reused as an index. I guess it wasn't affecting the `i` in the outer scope, because I think the vector was still being filled, but it seems like bad practice.